client/webserver/webserver.go

```diff
@@ -268,6 +269,7 @@ func New(core clientCore, addr string, logger dex.Logger, reloadHTML bool) (*Web
 			apiAuth.Post("/withdraw", s.apiWithdraw)
 			apiAuth.Post("/maxbuy", s.apiMaxBuy)
 			apiAuth.Post("/maxsell", s.apiMaxSell)
+			apiAuth.Post("/accountKeys", s.apiAccountKeys)
```

Comment:
Seeing as it is under the `rejectUnauthed`, I suppose we can eliminate the password argument to the Core method.